src/components/TodoListItem.js

```diff
@@ -2,13 +2,14 @@ import React from "react";
 import PropTypes from "prop-types";
 import style from "./TodoListItem.module.css";
 
-function TodoListItem({ todo, onRemoveTodo, isChecked, handleChange }) {
+function TodoListItem({ todo, onRemoveTodo, idChecked, handleChange }) {
   const onRemoveClick = () => {
     onRemoveTodo(todo.id);
   };
 
   const handleCheckboxChange = () => {
-    handleChange(todo.id);
+    let done = document.getElementById(`checkbox-${todo.id}`).checked;
+    handleChange(done);
   };
```

Comment:
You can get this value much more easily (and more "conventionally") by adding `event` parameter to this function (which is always passed automatically) and using `event.target.checked`

```suggestion
const handleCheckboxChange = (event) => {
    handleChange(event.target.checked);
};
```

Reply:
Thanks Darga! I edited it further to include     handleChange(todo.id, event.target.checked);
to pass along necessary info